src/uct/ib/mlx5/dv/ib_mlx5_dv.c

```diff
@@ -306,22 +300,89 @@ ucs_status_t uct_ib_mlx5_devx_modify_qp_state(uct_ib_mlx5_qp_t *qp,
 
 void uct_ib_mlx5_devx_destroy_qp(uct_ib_mlx5_md_t *md, uct_ib_mlx5_qp_t *qp)
 {
-    int ret = mlx5dv_devx_obj_destroy(qp->devx.obj);
-    if (ret) {
-        ucs_error("mlx5dv_devx_obj_destroy(QP) failed: %m");
-    }
+    uct_ib_mlx5_devx_obj_destroy(qp->devx.obj, "QP");
     uct_ib_mlx5_put_dbrec(qp->devx.dbrec);
     uct_ib_mlx5_md_buf_free(md, qp->devx.wq_buf, &qp->devx.mem);
 }
 
+ucs_status_t uct_ib_mlx5_devx_obj_modify(struct mlx5dv_devx_obj *obj,
+                                         const void *in, size_t inlen,
+                                         void *out, size_t outlen,
+                                         char *msg_arg)
+{
+    int ret           = mlx5dv_devx_obj_modify(obj, in, inlen, out, outlen);
+    unsigned syndrome = UCT_IB_MLX5DV_GET(general_obj_out_cmd_hdr, out,
+                                          syndrome);
+
+    if (ret) {
+        ucs_error("mlx5dv_devx_obj_modify(%s) failed, syndrome %x: %m",
```

Comment:
print also the device name. so may need to pass uct_ib_device_t or uct_ib_iface_t as a parameter.
syndrome 0x%x.

Reply:
let's think how can pass device name in next PR. maybe just pass name , or "struct ibv_device *"

Reply:
Honestly we can get ibv_context from mlx5dv_devx_obj - it's defined as 
```
struct mlx5dv_devx_obj {
        struct ibv_context *context;
        ...
};
```
Though this definition is not exported.

Reply:
Great idea, thank you!